### DISCUSSION TITLE
[Feature request] Generate readme.md for connector certification

### DISCUSSION THREAD

**Discussion by Laskewitz** [microsoft/powerplatform-vscode#589] [Ideas]
For certified and independent publisher connectors, we need to add a readme.md with details about the connector. Most of the connector details are already in the API Definition file, so it would be great to have a command that would generate the readme.md from the details that are in the API Definition file.

Here is an example of a readme.md: https://github.com/microsoft/PowerPlatformConnectors/blob/dev/independent-publisher-connectors/Discord/readme.md

The proposed command could be something like:
`pac connector markdown-create --api-definition ./apidefinition.swagger.json --output-file ./readme.md`

**Comment by troystaylor** [comment]
This is an interesting idea. I currently use swagger-markdown (https://www.npmjs.com/package/swagger-markdown) after downloading the connector, which lists the operations and operation descriptions. I think copy those values into a template readme.md file that I save into the connector folder.
